src/raftstore/store/config.rs

```diff
@@ -152,6 +152,14 @@ impl Config {
     }
 
     pub fn validate(&self) -> Result<()> {
+        if self.raft_heartbeat_ticks == 0 {
+            return Err(box_err!("heartbeat tick must greater than 0"));
+        }
+
+        if self.raft_election_timeout_ticks <= self.raft_heartbeat_ticks {
```

Comment:
do we allow equal here? 
Election should be more several times bigger than heartbeat, maybe we can define a max multiple like 5.